rs/nns/governance/src/governance/merge_neurons.rs

```diff
@@ -349,20 +349,28 @@ pub fn build_merge_neurons_response(
     now_seconds: u64,
     requester: PrincipalId,
 ) -> MergeResponse {
-    let source_neuron = Some(
+    let source_neuron = Some(api::Neuron::from(
```

Comment:
MergeNeuron is actually a sub-request within ManageNeuron (as can be seen by the first change in this file). This explains why we suddenly have to convert everything to api types in here. In the future, instead of having these conversions, into_proto would be replaced with into_api. That will be done in a later surgery PR.